integration_tests/dbt_project.yml

```diff
@@ -46,15 +46,15 @@ seeds:
   intercom_integration_tests:
     admin_data:
       +column_types:
-        id: "{{ 'int64' if target.name == 'bigquery' else 'bigint' }}"
+        id: "{{ 'varchar(100)'  if target.type == 'redshift' else 'string'}}"
```

Comment:
why are we doing this? is `target.name` the wrong attribute to reference?

Reply:
This probably did not need to be changed for our circle seed data set up. I had a similar snipped of code in a staging model of the initial release of the QuickBooks package and got customer feedback that models were failing because they had their taget.name something completely different from what we expect ('bigquery', 'snowflake', 'redshift'). After changing it to target.type the code ran for both of us. I started to make those changes, but since our target.name here is what we expect it probably is not necessary.